ocs_ci/framework/pytest_customization/ocscilib.py

```diff
@@ -539,6 +540,45 @@ def process_cluster_cli_params(config):
     cluster_path = os.path.expanduser(cluster_path)
     if not os.path.exists(cluster_path):
         os.makedirs(cluster_path)
+
+    # create kubconfig if doesn't exists and OCP url and kubeadmin password is provided
+    kubeconfig_path = os.path.join(
+        cluster_path, ocsci_config.RUN["kubeconfig_location"]
+    )
+    if not os.path.isfile(kubeconfig_path) and not get_cli_param(
+        config, "deploy", default=False
+    ):
+        if os.environ.get("KUBEADMIN_PASSWORD") and os.environ.get("OCP_URL"):
```

Comment:
can we have this option passed rather via config value?

I see an issue with multicluster execution when we will need multiple clusters like this it's not simply possible and better to have this value taken from config per specific cluster I think.

Also we will probably need to iterate over every cluster to get this.

Reply:
Good point, I was taking this approach as the easiest one for integration in jenkins, but you are right that for multicluster scenario this will not work and both creating config for that and loading it from it will be easy and directly multicluster ready.